### PR TITLE
Migration: add dependabot.yml

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes https://github.com/meilisearch/documentation/issues/2317

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  open-pull-requests-limit: 10
+  versioning-strategy: increase
+  rebase-strategy: disabled
```
Comment (guimachiavelli) [inline .github/dependabot.yml:10]:
```suggestion
  rebase-strategy: disabled

```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,927 @@
+{
+  "name": "documentation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
```
Comment (guimachiavelli) [inline package.json:6]: Ok, this is definitely not good. We're listing a few hundred dependencies, when we actually have none.

Delete this file, delete the `node_modules` dir on your local machine, and try running `npm init` again.

Comment (maryamsulemani97) [reply package.json:6]: The new file has the same dependencies as this one

Comment (guimachiavelli) [reply package.json:6]: hmmm…that's weird. Doing this locally seems to work on my machine.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,927 @@
...
+    "xmlbuilder": "^13.0.2",
+    "yargs-parser": "^13.1.2",
+    "zepto": "^1.2.0",
+    "yallist": "^3.1.1",
+    "zooming": "^2.1.1"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (guimachiavelli) [inline package.json:915]:
```suggestion
  "scripts": {},
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,927 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/meilisearch/documentation.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (guimachiavelli) [inline package.json:922]: Is our license ISC? I thought it was MIT? Please check the LICENSE file and update this.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,927 @@
...
+  "bugs": {
+    "url": "https://github.com/meilisearch/documentation/issues"
+  },
+  "homepage": "https://github.com/meilisearch/documentation#readme"
```
Comment (guimachiavelli) [inline package.json:926]:
```suggestion
  "homepage": "https://meilisearch.com/docs"
```

File: package.json (hunk 4 above)
Comment (maryamsulemani97) [reply package.json:14]: It's MIT

**Review by guimachiavelli — APPROVED** [review]
LGTM

**Pull request merged**